axiom/nr_php_packages.h

```diff
@@ -58,7 +67,8 @@ extern nr_php_package_t* nr_php_package_create_with_source(char* name, char* ver
  *           nr_php_packages_add_package() is not called, then it must be freed
  *           by nr_php_package_destroy()
  */
-extern nr_php_package_t* nr_php_package_create(char* name, char* version);
+extern nr_php_package_t* nr_php_package_create(const char* name,
+                                               const char* version);
```

Comment:
Formatting only, right?

Reply:
No I changed the pointers to be `const` as I saw no reason for them to not be.
Then `git-clang-format` did the rest.